src/components/inputs/RadioButton/RadioButton.scss

```diff
@@ -0,0 +1,81 @@
+$C: #{$WEBEX_COMPONENTS_CLASS_PREFIX}-radio-button;
+
+.#{$C} {
+  position: relative;
+  display: flex;
+
+  input[type=radio] {
+    all:unset;
+  }
+
+  input[type=radio]:before {
+    content: '';
+    box-sizing: border-box;
+    border-radius: 50%;
+    display: inline-block;
+    width: 16px;
+    height: 16px;
+  }
+
+  input[type=radio]:after {
+    content: '';
+    border-radius: 50%;
+    display: inline-block;
+    width: 6px;
+    height: 6px;
+    position: absolute;
+    top:5.5px;
+    left:5.5px;
+  }
+
+  &--enabled {
+    cursor:pointer;
+    
+    input[type=radio]:checked {
+      &:after {
+        background-color: var(--wxc-input-on--normal--foreground);
+      }
+      &:before {
+        background-color: var(--wxc-input-on--normal--background);
+      }
+      &:hover:before {
+        background-color: var(--wxc-input-on--hover--background);
+      }
+      &:active:before {
+        background-color: var(--wxc-input-on--active--background);
+      }
+    }
+
+    input[type=radio]:not(:checked) {
+      &:before {
+        background-color: var(--wxc-input-off--normal--background);
+        border: 0.063rem solid var(--wxc-input-off--normal--border-color);
```

Comment:
please put borders on all states, otherwise the control will have different dimensions when on or off. The border color should be the same as the background for some states.